code/game/turfs/open.dm

```diff
@@ -168,6 +168,7 @@
 /turf/open/river
 	can_bloody = FALSE
 	supports_surgery = FALSE
+	var/toxic = 0
```

Comment:
this won't be doing anything, as the toxic water behavior only exists in update_icon of /turf/open/desert/desert_shore, /turf/open/desert/cave/cave_shore and /turf/open/gm/river/desert. you need to remove the set_light(1) in all these cases

Reply:
i tryied by just placing "set_light = 0" but it give error back instead